sentry-slog/src/converters.rs

```diff
@@ -0,0 +1,87 @@
+use sentry_core::protocol::{Breadcrumb, Event, Exception, Frame, Level, Map, Stacktrace, Value};
+use slog::{OwnedKVList, Record, KV};
+
+/// Converts a `slog::Level` to a sentry `Level`
+pub fn convert_log_level(level: slog::Level) -> Level {
+    match level {
+        slog::Level::Trace | slog::Level::Debug => Level::Debug,
+        slog::Level::Info => Level::Info,
+        slog::Level::Warning => Level::Warning,
+        slog::Level::Error | slog::Level::Critical => Level::Error,
+    }
+}
+
+/// Adds the data from a `slog::KV` into a sentry `Map`.
+pub fn add_kv_to_map(map: &mut Map<String, Value>, kv: &impl KV) {
+    let _ = (map, kv);
+    // TODO: actually implement this ;-)
+}
+
+/// Creates a sentry `Breadcrumb` from the `slog::Record`.
```

Comment:
Hm, I will take a look and see what rustdoc actually generates, since this is cross-crates… In general, https://github.com/rust-lang/rust/issues/43466 is still nightly only :-(